.github/workflows/publish_v3.yml

```
@@ -32,12 +34,20 @@ jobs:
         run: pip install -r spec-parser/requirements.txt
       - name: Build model
         run: python3 spec-parser/main.py spdx-3-model/model spdx-spec/docs/model
+      - name: Generate JSON schema
+        run: |
+          shacl2code generate \
+            --input spdx-spec/docs/model/ontology.rdf.ttl \
+            --input spdx-spec/shacl2code-annotations.ttl \
+            --context-url spdx-spec/docs/model/context.jsonld https://spdx.github.io/spdx-spec/${{ env.VERSION }}/model/context.jsonld \
+            jsonschema \
+            --output spdx-spec/docs/model/schema.json
       - name: Set git identity
         run: git config user.name ci-bot; git config user.email ci-bot@spdx.dev
         working-directory: spdx-spec
       - name: Sync gh-pages
-        run: git checkout gh-pages && git pull && git checkout development/v3.0
+        run: git checkout gh-pages && git pull && git checkout development/${{ env.VERSION }}
```

This is messy code and a cleaner way of doing that might be
```suggestion
        run: |
          git fetch origin
          git branch -f gh-pages origin/gh-pages
```

---

as `development/v3.0` is hardcoded in line 18, making it dependent on the variable here causes inconsistencies

---

Reverted to hardcoded since it's inconsistent